ui/src/layouts/files.rs

```diff
@@ -52,6 +54,7 @@ pub fn FilesLayout(cx: Scope<Props>) -> Element {
                                 Button {
                                     icon: Icon::FolderPlus,
                                     appearance: Appearance::Secondary,
+                                    aria_label: "add-folder".into(),
```

Comment:
Have you made sure this displays as expected? Only reason I ask is because Button is a custom element and IDK if it has the aria_label prop (unless you added that previously.)

Reply:
Yes, I added the aria_label as a prop in the UI kit before. Here is a screenshot on how it would be displayed:

![image](https://user-images.githubusercontent.com/35935591/211429356-4b9f37c5-90df-49d7-8466-b26dd9d31227.png)